src/app/child-dev-project/reporting/reporting/report-row/report-row.component.ts

```diff
@@ -0,0 +1,25 @@
+import { Component, Input, OnInit } from "@angular/core";
+import { ReportRow } from "../../reporting.service";
+
+@Component({
+  selector: "app-report-row",
+  templateUrl: "./report-row.component.html",
+  styleUrls: ["./report-row.component.scss"],
+})
+export class ReportRowComponent implements OnInit {
+  @Input() rows: ReportRow[] = [];
+  @Input() values: string[];
```

Comment:
can you add jsdoc comments here to all inputs? It's not fully clear what these `values` are